### PR TITLE
Fwdport 6.2 to 6.2.x

### PR DESCRIPTION
**Pull request by alexandre-merle**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,15 +11,15 @@
   },
   "homepage": "https://github.com/scality/vaultclient#readme",
   "dependencies": {
-    "arsenal": "scality/Arsenal",
+    "arsenal": "scality/Arsenal#rel/6.2.1",
```
Comment (DavidPineauScality) [inline package.json]: let's be careful :) Please revert the dependency change here

Comment (alexandre-merle) [inline package.json]: Sorry, the PR is not good target :) my bad

**Comment by DavidPineauScality** [comment]
E2E launched globally from https://github.com/scality/Integration/pull/287

**Comment by DavidPineauScality** [comment]
Tests are passing, exceptionally merging to enforce progress on this subject.

**Pull request merged**